packages/data/README.md

```diff
@@ -275,19 +275,19 @@ _Usage_
 import { useSelect, AsyncModeProvider } from '@wordpress/data';
 
 function BlockCount() {
-	const count = useSelect( ( select ) => {
-		return select( 'core/block-editor' ).getBlockCount();
-	}, [] );
+  const count = useSelect( ( select ) => {
```

Comment:
Nit: it looks like you formatted all markdown files using a different tool or Prettier config.